lib/Lift.cpp

```diff
@@ -590,21 +590,39 @@ bool LiftCodeIntoModule(const remill::Arch *arch, const Program &program,
 
   // Lift global variables.
   program.ForEachVariable([&](const anvill::GlobalVarDecl *decl) {
-    const auto addr = decl->address;
-    const auto name = anvill::CreateVariableName(addr);
-    const auto gvar = decl->DeclareInModule(name, module);
+    const auto addr{decl->address};
+    const auto name{anvill::CreateVariableName(addr)};
+    const auto gvar{decl->DeclareInModule(name, module)};
 
+    // Check if we have mapped bytes
+    if (!program.FindByte(addr)) {
+      return true;
+    }
     // Set initializer
-    auto init = CreateConstFromMemory(addr, decl->type, arch, program, module);
+    auto init{CreateConstFromMemory(addr, decl->type, arch, program, module)};
     gvar->setInitializer(init);
-
     return true;
   });
 
   // Lift functions.
   program.ForEachFunction([&](const FunctionDecl *decl) {
-    const auto entry = lifter.LiftFunction(*decl);
-    DefineNativeToLiftedWrapper(arch, *decl, entry);
+    // Initialize function entry
+    const auto name{CreateFunctionName(decl->address)};
+    FunctionEntry entry{nullptr, nullptr, nullptr};
+    entry.lifted_to_native =
```

Comment:
What is the `lifted_to_native` function? I am assuming it's used when you pass this function's address to callbacks?

Reply:
You should look at the comments in `FunctionEntry`, they explain it well.